jenkins_jobs/run_host_os_admin_commands.groovy

```diff
@@ -0,0 +1,39 @@
+job('run_host_os_admin_commands') {
+  label('builds_slave_label')
+  logRotator {
+    numToKeep(30)
+  }
+  concurrentBuild()
+  throttleConcurrentBuilds {
+    maxPerNode(1)
+  }
```

Comment:
Since those commands are pretty fast to execute, we can probably allow concurrent builds in the same node. You can remove `throttleConcurrentBuilds`